sui/README.md

```diff
@@ -85,137 +102,91 @@ node sui/deploy-contract.js deploy AxelarGateway --signers wallet --nonce test
 node sui/deploy-contract.js deploy AxelarGateway -e testnet --signers '{"signers": [{"pub_key": "0x020194ead85b350d90472117e6122cf1764d93bf17d6de4b51b03d19afc4d6302b", "weight": 1}], "threshold": 1, "nonce": "0x0000000000000000000000000000000000000000000000000000000000000000"}'
 ```
 
-Upgrading Gateway:
-
-To update the gateway run the following command:
-
-```bash
-node sui/deploy-contract.js upgrade AxelarGateway <policy>
-```
-
-policy should be one of the following:
-
--   `any_upgrade`: Allow any upgrade.
--   `code_upgrade`: Upgrade policy to just add code. https://docs.sui.io/references/framework/sui-framework/package#function-only_additive_upgrades
--   `dep_upgrade`: Upgrade policy to just change dependencies. https://docs.sui.io/references/framework/sui-framework/package#function-only_dep_upgrades
-
-Provide `--txFilePath` with `--offline` to generate tx data file for offline signing.
-
-Deploy the Utils package:
-
-```bash
-node sui/deploy-contract.js deploy Utils
-```
-
-Deploy the Gas Service package:
+### Gas Service
 
 ```bash
 node sui/deploy-contract.js deploy GasService
 ```
 
-Deploy the test GMP package:
+### Abi
 
 ```bash
-node sui/deploy-contract.js deploy Example
+node sui/deploy-contract.js deploy Abi
 ```
 
-Deploy the Operators package:
+### Operators
 
 ```bash
 node sui/deploy-contract.js deploy Operators
 ```
```

Comment:
add the post deployment step for each contract right after instead of having a separate section